046_App_L245/046_App_L245_safe.ts

```
@@ -0,0 +1,299 @@
+      resave: true,
+      saveUninitialized: true }));
+    this.expressApp.use(cookieParser());
+    this.expressApp.use(passport.initialize());
+    this.expressApp.use(passport.session());
+  }
+
+  // Validates if the user is authenticated for an API
+  private validateAuth(req:any, res:any, next:any):void {
+    if (req.isAuthenticated()) { 
+      console.log("user is authenticated" );
+      return next(); 
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)